src/Validation/Validation.php

```diff
@@ -308,6 +308,24 @@ public static function compareWith($check, $field, $context)
     }
 
     /**
+     * Checks if a string contains one or more non-alphanumeric characters.
+     *
+     * Returns true if string contains at least the specified number of non-alphanumeric characters
+     *
+     * @param string $check Value to check
+     * @param int $count Number of non-alphanumerics to check for
+     * @return bool Success
+     */
+    public static function containsNonAlphaNumeric($check, $count = 1)
+    {
+        preg_match_all('/[^a-zA-Z0-9]/', $check, $matches);
```

Comment:
The whole method can be replaced by:

```php
$regex = '/[^a-zA-Z0-9]/';
return self::_check($check, $regex);
```

_check() includes is_string() check.